spec/plans/results.fmf

```diff
@@ -8,8 +8,12 @@ description: |
     and custom test results are required to follow it when creating their
     ``results.yaml`` file.
 
-    Results are saved as a YAML file, containing a single list of mappings,
-    one mapping describing a single test result.
+    Tests may choose JSON instead of YAML for their custom results file and
+    create ``results.json`` file, but tmt itself will always stick to YAML,
+    the final results would be provided in ``results.yaml`` file in any case.
+
+    Results are saved as a single list of mappings, one mapping describing
```

Comment:
Could we use `dictionary` instead of `mapping` to keep terminology close to Python? Or... use `sequence` instead of `list` to keep `yaml` terminology? I'd prefer a bit more the first one, but let's be consistent.

Reply:
Updated in 9cea19ca.